### ISSUE TITLE
Getting error of Error while loading rule 'vue/attribute-hyphenation': Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by suil** [vuejs/eslint-plugin-vue#285]
**Tell us about your environment**

* **ESLint Version:** 4.12.1
* **eslint-plugin-vue Version:** 4.0.0-beta.2
* **Node Version:** v8.9.1

**Please show your full configuration:**

```
// http://eslint.org/docs/user-guide/configuring

module.exports = {
    root: true,
    parser: 'babel-eslint',
    parserOptions: {
        sourceType: 'module'
    },
    env: {
        browser: true,
    },
    extends: [
        'airbnb-base',
        'plugin:vue/recommended',
    ],
    // required to lint *.vue files
    plugins: [
        'html'
    ],
    // check if imports actually resolve
    'settings': {
        'import/resolver': {
            'webpack': {
                'config': 'build/webpack.base.conf.js'
            }
        }
    },
    // add your custom rules here
    'rules': {
        // don't require .vue extension when importing
        'import/extensions': ['error', 'always', {
            'js': 'never',
            'vue': 'never',
            'svg': 'never'
        }],
        // allow optionalDependencies
        'import/no-extraneous-dependencies': ['error', {
            'optionalDependencies': ['test/unit/index.js']
        }],
        // allow debugger during development
        'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
        // indentation is 4 spaces
        'indent': ['error', 4, {SwitchCase: 1}],
        // disable warning for different OS line ends
        'linebreak-style': 0,
        'comma-dangle': ['error', {
            'arrays': 'only-multiline',
            'objects': 'only-multiline',
            'imports': 'never',
            'exports': 'never',
            'functions': 'ignore'
        }],
        'space-before-function-paren': ['error', {
            'anonymous': 'always',
            'named': 'never',
            'asyncArrow': 'always'
        }],
        "function-paren-newline": ["error", "multiline"],
        "arrow-parens": ["error", "as-needed"],
        "max-len": [1, process.env.NODE_ENV === 'test' ? 300 : 120, 4, {"ignoreUrls": true}],
        "vue/attribute-hyphenation": [2, 'always'|'never']
    }
};

```

**What did you do? Please include the actual source code causing the issue.**

```vue





```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**
```
Module build failed: TypeError: Error while loading rule 'vue/attribute-hyphenation': Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at Object.keys.forEach.ruleId (D:\Projects\mds-vue\node_modules\eslint\lib\linter.js:928:24)
    at Array.forEach (<anonymous>)
    at Linter._verifyWithoutProcessors (D:\Projects\mds-vue\node_modules\eslint\lib\linter.js:865:35)
    at preprocess.map.textBlock (D:\Projects\mds-vue\node_modules\eslint\lib\linter.js:994:35)
    at Array.map (<anonymous>)
    at Linter.verify (D:\Projects\mds-vue\node_modules\eslint\lib\linter.js:993:42)
    at localVerify (D:\Projects\mds-vue\node_modules\eslint-plugin-html\src\index.js:107:14)
    at Linter.eslint.verify (D:\Projects\mds-vue\node_modules\eslint-plugin-html\src\index.js:147:18)
    at Linter.verifyAndFix (D:\Projects\mds-vue\node_modules\eslint\lib\linter.js:1085:29)
    at processText (D:\Projects\mds-vue\node_modules\eslint\lib\cli-engine.js:179:32)
    at CLIEngine.executeOnText (D:\Projects\mds-vue\node_modules\eslint\lib\cli-engine.js:614:17)
    at lint (D:\Projects\mds-vue\node_modules\eslint-loader\index.js:218:17)
    at Object.module.exports (D:\Projects\mds-vue\node_modules\eslint-loader\index.js:213:21)
```

**Comment by mysticatea** [comment]
Thank you for this issue.

I think that this is a duplicate of #259, #261 and #283.
